Fixtures/Miscellaneous/HangingTest/Tests/HangingTestTests/HangingTestTests.swift

```diff
@@ -0,0 +1,15 @@
+#if os(Windows)
+import CRT
+#elseif canImport(Darwin)
+import Darwin
+#elseif canImport(Glibc)
+import Glibc
+#endif
+
+final class aaaaTests: XCTestCase {
+    func testExample() throws {
+        while true {
+            sleep(1)
```

Comment:
@compnerd not sure if `sleep` is available on Windows in CRT?

Reply:
It is not.  Ideally, you would just use `Foundation.Thread.sleep(for:)`.  Windows has a `Sleep` that is part of the Windows SDK that takes time in ms.

Reply:
Yes, that's obvious in hindsight to me now, especially as I'm already using this function in the test that exercises this fixture 😅